include/rogue/EnableSharedFromThis.h

```diff
@@ -0,0 +1,41 @@
+/**
+ *-----------------------------------------------------------------------------
+ * Title      : Stream memory pool
```

Comment:
This code header looks correct.  This is not the `Pool.h`

Reply:
Yes good catch. I meant to go back and fix that after I verified it solved my problem.

Reply:
Fixed now.